packages/client/src/sharedWishlists/__tests__/deleteSharedWishlist.test.ts

```diff
@@ -0,0 +1,38 @@
+import { deleteSharedWishlist } from '..';
+import { mockSharedWishlistId } from 'tests/__fixtures__/sharedWishlists';
+import client from '../../helpers/client';
+import fixtures from '../__fixtures__/deleteSharedWishlist.fixtures';
+import mswServer from '../../../tests/mswServer';
+
+describe('deleteSharedWishlist', () => {
+  const expectedConfig = undefined;
+  const spy = jest.spyOn(client, 'delete');
+
+  beforeEach(jest.clearAllMocks);
+
+  it('should handle a client request successfully', async () => {
+    mswServer.use(fixtures.success());
+    expect.assertions(2);
```

Comment:
No need for the `expect.assertions` here as you are awaiting the promise already. If it fails, an error will occur and the test will fail.